components/camel-infinispan/camel-infinispan-embedded/pom.xml

```diff
@@ -98,12 +98,12 @@
             <version>${infinispan-version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-spring5-embedded</artifactId>
-            <version>${infinispan-version}</version>
-            <scope>test</scope>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
Maybe this can be removed

Reply:
Yeah, I should have removed it. It should be clear now.